bundles/org.openhab.binding.miio/src/main/resources/database/cgllc.airm.cgdn1-miot.json

```diff
@@ -0,0 +1,492 @@
+				"siid": 3,
+				"piid": 5,
+				"friendlyName": "Environment - PM10 Density",
+				"channel": "pm10_density",
+				"type": "Number",
+				"unit": "μg/m3",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 1000,
+					"step": 1,
+					"pattern": "%.1f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "temperature",
+				"siid": 3,
+				"piid": 7,
+				"friendlyName": "Environment - Temperature",
+				"channel": "temperature",
+				"type": "Number:Temperature",
+				"unit": "celsius",
+				"stateDescription": {
+					"minimum": -30,
+					"maximum": 100,
+					"pattern": "%.1f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "co2-density",
+				"siid": 3,
+				"piid": 8,
+				"friendlyName": "Environment - CO2 Density",
+				"channel": "co2_density",
+				"type": "Number:Dimensionless",
+				"unit": "ppm",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 9999,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "battery-level",
+				"siid": 4,
+				"piid": 1,
+				"friendlyName": "Battery - Battery Level",
+				"channel": "battery_level",
+				"type": "Number:Dimensionless",
+				"unit": "percentage",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 100,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "charging-state",
+				"siid": 4,
+				"piid": 2,
+				"friendlyName": "Battery - Charging State",
+				"channel": "charging_state",
+				"type": "Number",
+				"stateDescription": {
+					"readOnly": true,
+					"options": [
+						{
+							"value": "1",
+							"label": "Charging"
+						},
+						{
+							"value": "2",
+							"label": "Not charging"
+						},
+						{
+							"value": "3",
+							"label": "Not chargeable"
+						}
+					]
+				},
+				"refresh": true,
+				"actions": [],
+				"readmeComment": "Value mapping [\"1\"\u003d\"Charging\",\"2\"\u003d\"Not charging\",\"3\"\u003d\"Not chargeable\"]"
+			},
+			{
+				"property": "voltage",
+				"siid": 4,
+				"piid": 3,
+				"friendlyName": "Battery - Voltage",
+				"channel": "voltage",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 65535,
+					"step": 1,
+					"pattern": "%.1f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "mac",
+				"siid": 8,
+				"piid": 1,
+				"friendlyName": "Mac - Mac",
+				"channel": "mac",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "start-time",
+				"siid": 9,
+				"piid": 2,
+				"friendlyName": "Settings - Start Time",
+				"channel": "start_time",
+				"type": "Number",
```

Comment:
Is this supposed to be a time unit?

Reply:
@joeberlin123 can you confirm what you are seeing for this channel as value and how it translates in the mihome app.
Than we can determine what is indeed it is (e.g. the time until it switches on/off or a date/time sort of thing.

The spec does not hint anything

Reply:
I don't find something like start time in the mi home app. If I use item type number I get 0.0 and if I use date time I get NULL. Do have another hint for me?

Reply:
I took out this property... if we can't find it in the app and it is unknown what it does, it is useless.